### DISCUSSION TITLE
MHIG_RW_VW_S1538 Patchin Help to fix Nav

### DISCUSSION THREAD

**Discussion by immy906k** [Mr-MIBonk/M.I.B._More-Incredible-Bash#599] [Q&A]
Hi All 
i broke my navigation by downloading 2024 ROW maps from vw official site and updating via sd card.
system had fec 091d0002 installed with 2014/2015 ROW maps.

any help on how to fix? i tried patching with MIB.

![20240910_180235](https://github.com/user-attachments/assets/27847e16-6931-408f-8e39-df3e6354c711)
![20240910_180244](https://github.com/user-attachments/assets/e0fb43cc-21fc-4aa9-a29d-fa3ca47a24eb)
![20241119_170647](https://github.com/user-attachments/assets/006bb89c-0116-4db0-851b-5bbd187a9160)
![20241119_170658](https://github.com/user-attachments/assets/2befe97b-e062-4921-9d15-062d44438049)
![20241119_170732](https://github.com/user-attachments/assets/826042a6-6b33-434d-a8b8-0877354d6398)
![20240910_170852](https://github.com/user-attachments/assets/9948c711-3144-4ffc-a874-309152124714)
![20240910_170902](https://github.com/user-attachments/assets/9349587d-3206-479c-98e6-2751a6323d35)
![20240910_180301](https://github.com/user-attachments/assets/fc3f7dbb-dee2-469c-8858-3b80df89c142)
![20241031_174240](https://github.com/user-attachments/assets/0596bd10-2a0e-493d-a61c-95ddcff9b0ae)
![20241101_203159](https://github.com/user-attachments/assets/e9ee0dad-f4da-4a81-b637-fc520e979e9c)
![20241119_170527](https://github.com/user-attachments/assets/574acf96-fda8-4ef4-b088-11599716066b)
![20241119_170621](https://github.com/user-attachments/assets/4e2339c6-5d8e-4328-ab29-b6f7c23e97ee)
![20241119_170709](https://github.com/user-attachments/assets/f2af5ba3-45a7-4434-96e2-d47b81dfe69c)
![20241119_170720](https://github.com/user-attachments/assets/53eaa08c-afa4-469e-9e31-5460575d1dd1)
![20240910_180251](https://github.com/user-attachments/assets/43e8c57e-6d12-4023-94d7-1c1f991dd57c)
![20240910_180318](https://github.com/user-attachments/assets/b200f27b-cc5b-48a0-b68d-556305afd83c)
![20241119_170614](https://github.com/user-attachments/assets/020f0b8a-deff-49df-88c8-c0b6fa217c07)
![20241119_170634](https://github.com/user-attachments/assets/4893d5ef-fc61-46c2-9551-f6e658a86de1)

**Comment by Cloudmania4** [comment]
Fixed?

**Reply by immy906k** [reply]
Hi no, havent resolved this issue maps still broken.

Any direction on how to fix?

**Comment by Cloudmania4** [comment]
Enter telegram group and ask about mhig rw.

**Reply by immy906k** [reply]
Please send link if possible 🙏.  
Been trying to fix this with no luck for sometime now.

**Comment by Cloudmania4** [comment]
https://t.me/joinchat/EHt4RRksHcMQk6Xi6tFaBw
